src/Oro/ORM/Query/AST/Platform/Functions/Postgresql/Timestampdiff.php

```diff
@@ -90,7 +78,10 @@ protected function getDiffForMicrosecond(Node $firstDateNode, Node $secondDateNo
      */
     protected function getDiffForMinute(Node $firstDateNode, Node $secondDateNode, SqlWalker $sqlWalker)
     {
-        return $this->getDiffForSecond($firstDateNode, $secondDateNode, $sqlWalker) . ' / 60';
+        return $this->getRoundedValue(
+            $this->getDiffForSecond($firstDateNode, $secondDateNode, $sqlWalker) . ' / 60',
```

Comment:
This approach will give wrong result because of rounding e.g.:
`SELECT ROUND((EXTRACT(EPOCH FROM TIMESTAMP '2014-01-01 00:01:31') - EXTRACT(EPOCH FROM TIMESTAMP '2014-01-01 00:00:00')) / 60);`
will give **2** instead of expected **1**.